java-manta-it/src/test/java/com/joyent/manta/client/multipart/EncryptedServerSideMultipartManagerIT.java

```diff
@@ -431,6 +434,7 @@ public void cantUploadSmallMultipartString() throws IOException {
                                         });
             assert exception.getCause() instanceof IllegalStateException;
         }
+        Assert.assertEquals(uploadedParts.size(), 1, "only first small upload should succeed");
```

Comment:
I added this for an error that I got once and have not been able to reproduce.  I'm hopeful that this assertion will make me feel less crazy.